### DISCUSSION TITLE
What should be a clean way to validate password during the reset password router flow?

### DISCUSSION THREAD

**Discussion by satwikkansal** [fastapi-users/fastapi-users#465] [Q&A] [answered]
Hi,

First of all, thanks for this amazing library. The documentation is really good.

I was wondering if there's a FastAPIish way to validate passwords during the reset flow, because the request bypasses the validation defined in the pydantic modes (as specified in the docs [here](https://frankie567.github.io/fastapi-users/configuration/model/#password-validation))

**Comment by frankie567** [comment]
Hi @satwikkansal!

That's indeed a good question I didn't thought about... Currently, the reset password router just expects a string, so there is no way currently to validate it at this point.

I'll open an issue so that it can be solved! Thank you 😄

**Reply by frankie567** [reply]
See #474.

**Reply by satwikkansal** [reply]
Thanks @frankie567 , looking forward to the feature :)

**Comment by frankie567** [comment, marked as answer]
The feature is now live in v6.1.0 🎉 Read more here: https://frankie567.github.io/fastapi-users/configuration/password-validation/

**Reply by satwikkansal** [reply]
Thank you!
